tests/test_current_inventory.py

```diff
@@ -14,25 +13,31 @@ def make_beer(quantity):
 
 
 def make_queries(filename):
+    """Tests that a data file can be turned into an InventoryQuery"""
+    base_dir = 'sample_csv_files/'
     return InventoryQueries(Inventory(base_dir + filename))
 
 
 def test_no_inventory():
+    """Tests that no inventory file adds no inventory"""
     queries = make_queries('empty.csv')
     assert [] == queries.get_available_inventory()
 
 
 def test_single_item_with_positive_quantity():
+    """Tests that a file with a single beer adds one beer to inventory"""
     expected = [make_beer(1)]
     queries = make_queries('single_positive_quantity.csv')
     assert expected == queries.get_available_inventory()
 
 
 def test_single_item_with_zero_quantity():
+    """Tests that one beer type with no quantity will generate empty inventory"""
```

Comment:
Might be good to change the wording here to the same form as test_single_item_with_negative_quantity's docstring